src/routes/+page.svelte

```diff
@@ -25,34 +25,92 @@
     </div>
 </section>
 
-<section class="px-6 lg:px-32 pt-24">
+<section class="px-6 lg:px-32 pt-12 pb-8">
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
+        >
             <span class="pr-4"><Values /></span> Beliefs
         </h2>
-        <p class="text-xl font-interMedium max-w-[580px]">
+        <p class="text-2xl lg:text-3xl font-interRegular max-w-[720px]">
             Crypto has the potential to give people across the world almost
             unimaginable levels of freedom and autonomy.
         </p>
     </div>
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
+        >
             <span class="pr-4"><Values /></span> Vision
```

Comment:
Why there's an element called `<Values />` being placed along Vision and Beliefs

Reply:
That's an svg image component